src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -82,6 +82,7 @@ public async Task SetListenerTimeoutWorksWithSmallTimeout()
             await Assert.ThrowsAsync<QuicOperationAbortedException>(async () => await serverConnection.AcceptStreamAsync().TimeoutAfter(100000));
         }
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/49157")]
```

Comment:
We're not running these in CI AFAIK so the failures are not blocking any PRs. This will disable them for our local runs, is that what you're goal here is?

Reply:
Yep, it is for us to have a clean local test run, since we are to start adding new changes. So if some new failures appear, we will know it was because of our changes